include/benchmark/benchmark.h

```diff
@@ -442,100 +101,122 @@ class Benchmark {
   // Equivalent to ThreadRange(NumCPUs(), NumCPUs())
   Benchmark* ThreadPerCpu();
 
+  // TODO(sanjay): Control whether or not real-time is used for this benchmark
+  // TODO(sanjay): Control the default number of iterations
+
   // -------------------------------
   // Following methods are not useful for clients
 
   // Used inside the benchmark implementation
   struct Instance;
 
-  // Measure the overhead of an empty benchmark to subtract later.
-  static void MeasureOverhead();
+  // Extract the list of benchmark instances that match the specified
+  // regular expression.
+  static void FindBenchmarks(const std::string& re,
```

Comment:
I moved it into the internal namespace because that is how main does it. However I'm happy moving this out of the headers entirely.